tests/e2e/test_unset.py

```
@@ -0,0 +1,71 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)